interop-tests/chromium-ping-version.json

```
@@ -1,7 +1,10 @@
 {
-  "id": "chromium-rust-libp2p-head",
-  "containerImageID": "chromium-rust-libp2p-head",
-  "transports": [{ "name": "webtransport", "onlyDial": true }],
-  "secureChannels": [],
-  "muxers": []
+    "id": "chromium-rust-libp2p-head",
+    "containerImageID": "chromium-rust-libp2p-head",
+    "transports": [
+        { "name": "webtransport", "onlyDial": true },
+        { "name": "webrtc-websys", "onlyDial": true }
```

```suggestion
        { "name": "webrtc-direct", "onlyDial": true }
```

These are "standardized" in that the testrunner has specific knowledge about the ones that have a muxer built in, see https://github.com/libp2p/test-plans/blob/e8c0733ce59f3d5721c565dc8557681628e61a95/multidim-interop/src/generator.ts#L57.

---

Done

---

Done